### PR TITLE
fix: correct ID used for refetching after updating customer

### PR DESCRIPTION
**Pull request by oDestroyeRo**
What
   correct ID used for refetching after updating customer

### CODE REVIEW

File: integration-tests/modules/__tests__/customer/store/update-customer.spec.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { medusaIntegrationTestRunner } from "@medusajs/test-utils"
```
Comment (riqwan) [inline integration-tests/modules/__tests__/customer/store/update-customer.spec.ts]: Can you move this test to `integration-tests/http/__tests__/customer/store/customer.spec.ts` under a new describe i.e. what you have from line 15?

**Review by riqwan — APPROVED** [review]
Nice, thanks for the contribution @oDestroyeRo!

File: integration-tests/modules/__tests__/customer/store/update-customer.spec.ts (hunk 1 above)
Comment (oDestroyeRo) [reply integration-tests/modules/__tests__/customer/store/update-customer.spec.ts]: I’ve moved the test to integration-tests/http/__tests__/customer/store/customer.spec.ts

Comment (riqwan) [reply integration-tests/modules/__tests__/customer/store/update-customer.spec.ts]: legend! 💪🏻

**Pull request merged**